stories/__tests__/SQFormDatePicker.stories.test.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {LocalizationProvider} from '@material-ui/pickers';
+import MomentAdapter from '@material-ui/pickers/adapter/moment';
+import {composeStories} from '@storybook/testing-react';
+import {render, screen, within} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormDatePicker.stories';
+
+const {BasicDatePicker} = composeStories(stories);
+
+const renderDatePicker = props => {
+  render(
+    <LocalizationProvider dateAdapter={MomentAdapter} locale={'en'}>
+      <BasicDatePicker {...props} />
+    </LocalizationProvider>
+  );
+};
+
+describe('SQFormDatePicker Tests', () => {
+  it('should render a label and input', () => {
+    renderDatePicker();
+
+    const label = screen.getByText('Date');
+    expect(label).toBeInTheDocument();
+    expect(label).toHaveTextContent('Date');
```

Comment:
I think this `expect` is redundant -- you've used `getByText('Date')` to get the `label`, so it's safe to assume that it also has the text content `'Date'`